pkgs/development/tools/godot4/pkg_config_additions.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/platform/linuxbsd/detect.py b/platform/linuxbsd/detect.py
+index 1487210..394cb9e 100755
+--- a/platform/linuxbsd/detect.py
++++ b/platform/linuxbsd/detect.py
+@@ -214,6 +214,9 @@ def configure(env):
+     env.ParseConfig("pkg-config xrandr --cflags --libs")
+     env.ParseConfig("pkg-config xrender --cflags --libs")
+     env.ParseConfig("pkg-config xi --cflags --libs")
++    env.ParseConfig("pkg-config xfixes --cflags --libs")
++    env.ParseConfig("pkg-config glu --cflags --libs")
```

Comment:
Is this also intended for upstreaming? Looks like it could be.

Reply:
@Mindavi i just updated the patch file in the current godot package. I'm not exactly sure why the patch is needed or if it should be upstreamed.